src/solver/postprocess.jl

```diff
@@ -122,3 +135,33 @@ function postprocess_root(x)
     end
     x # unreachable
 end
+
+function check_trig_consts(x)
```

Comment:
Missing a docstring, and should add to the docs

Also, is the name correct? For example, why not add `e` to this?

Reply:
wdym e?

Reply:
Also adding this to the docs is a bit weird. Used only once and the user wouldnt really care. Some other functions like filter_poly are much more interesting. What do you think?

Reply:
A general function for taking floats to common symbolic constants has lots of other uses?

Reply:
its the reverse but yes ok makes sense

Reply:
This conversation was not resolved. Why was it merged?

Reply:
check the latest couple of commits, added a small docstring and made the array a const